sdk-commons/src/main/java/org/openmrs/maven/plugins/model/BaseSdkProperties.java

```diff
@@ -154,9 +154,14 @@ public List<Artifact> getContentArtifacts() {
 		List<Artifact> artifacts = new ArrayList<>();
```

Comment:
@mseaton wouldn't it be better to consolidate duplicated code for extracting different artifact types into a single reusable method?

```suggestion
    public List<Artifact> getWarArtifacts(){
        return getArtifacts(TYPE_WAR);
    }

	public List<Artifact> getConfigArtifacts() {
        return getArtifacts(TYPE_CONFIG);
    }

	public List<Artifact> getContentArtifacts() {
        return getArtifacts(TYPE_CONTENT);
    }

    private List<Artifact> getArtifacts(String typeContent) {
        List<Artifact> artifacts = new ArrayList<>();
        for (Object keyObject : getAllKeys()) {
            String key = keyObject.toString();
String artifactType = getArtifactType(key);
            if (artifactType.equals(typeContent)) {
                artifacts.add(new Artifact(
checkIfOverwritten(key, ARTIFACT_ID),
getParam(key),
                        checkIfOverwritten(key, GROUP_ID),
checkIfOverwritten(key, TYPE)
));
            }
        }
        return artifacts;
    
```

Reply:
Indeed @wikumChamith there are many, many opportunities to work on tech debt in the SDK.  This initial PR is attempting to just fix the underlying issue.  More than happy to make broader refactoring improvements in other commits / tickets.